docs/guides/node-operator-manual.md

```diff
@@ -105,23 +105,43 @@ a Node Operator submits a set of the corresponding signatures [as defined in the
 The fork version used for generating the signature must correspond to the fork version of the Beacon
 chain the instance of Lido protocol is targeted to.
 
+#### Mainnet
+
+Make sure to obtain a correct withdrawal address by finding it inside active withdrawal credentials either on Aragon UI or calling the contract via [`Lido.getWithdrawalCredentials()`]. You can quickly find on the [Etherscan page for the Mainnet-deployed Lido].
```

Comment:
inside _the_ active ...
You can find the method on the ... (no `quickly`, add `method`)